src/modules/dashboard/components/Extensions/ExtensionUninstallConfirmDialog/ExtensionUninstallConfirmDialog.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ReactNode, useState } from 'react';
+import { MessageDescriptor, useIntl } from 'react-intl';
+
+import Button from '~core/Button';
+import Heading from '~core/Heading';
+import Dialog from '~core/Dialog';
+import DialogSection from '~core/Dialog/DialogSection';
+import InputLabel from '~core/Fields/InputLabel';
+
+import { ExtensionsMSG } from '../extensionsMSG';
+import styles from './ExtensionUninstallConfirmDialog.css';
+
+interface Props {
+  cancel: () => void;
+  close: (val: any) => void;
+  onClick?: () => void;
+  heading?: string | MessageDescriptor;
+  children?: ReactNode;
+}
+
+const ExtensionUninstallConfirmDialog = ({
+  cancel,
+  close,
+  heading,
+  children,
+  onClick = () => close(null),
+}: Props) => {
+  const { formatMessage } = useIntl();
+  const [isWarningInputValid, setIsWarningInputValid] = useState<boolean>(
+    false,
+  );
+
+  const onWarningInputChange = (e) => {
+    setIsWarningInputValid(e.target.value === 'I UNDERSTAND');
+  };
```

Comment:
I would argue that this should be a single word, not two.

So I would vote for something like "UNINSTALL". That way you can just copy/paste to uninstall it :)

Reply:
So you need to argue with product team :D